examples/ingest/s3-small-batch/ingest.sh

```diff
@@ -12,5 +12,9 @@ PYTHONPATH=. ./unstructured/ingest/main.py \
         s3 \
          --remote-url s3://utic-dev-tech-fixtures/small-pdf-set/ \
          --anonymous \
-         --structured-output-dir s3-small-batch-output \
-         --num-processes 2
+         --output-dir s3-small-batch-output \
+         --num-processes 2 \
+         --verbose \
+         s3 \
+         --anonymous \
+         --remote-url s3://utic-dev-tech-fixtures/small-pdf-set-output
```

Comment:
These are no different than the connector configs being used for the upstream connection, but used for the downstream one. It might look exceptionally weird here because we're reading and writing to s3 with the same configs but this won't always be the case.

Reply:
oh no, meant more broadly that the general form of the subcommand of s3 to specify output. appreciate the re-use of the configs here. i.e. just generally calling 

```
./unstructured/ingest/main.py s3 --some --options s3 --some --other -options
```

and tracking that this is input and output. though maybe making it clear in the docs and help is enough, have definitely worked through far more confusing conventions in clis.

Reply:
oh yea, totally appreciate the complexity and don't have any immediate thoughts.